curvefs/src/client/inode_cache_manager.cpp

```diff
@@ -341,5 +345,69 @@ void InodeCacheManagerImpl::ReleaseCache(uint64_t parentId) {
     iAttrCache_->Release(parentId);
 }
 
+void InodeCacheManagerImpl::FlushInodeBackground() {
+    LOG(INFO) << "flush thread is start.";
+    while (!isStop_.load()) {
+        if (iCache_->Size() > maxCacheSize_) {
+            TrimIcache(iCache_->Size() - maxCacheSize_);
+        }
+        sleeper_.wait_for(std::chrono::seconds(flushPeriodSec_));
+        FlushInodeOnce();
+    }
+    LOG(INFO) << "flush thread is stop.";
+}
+
+void InodeCacheManagerImpl::TrimIcache(uint64_t trimSize) {
+    std::shared_ptr<InodeWrapper> inodeWrapper;
+    uint64_t inodeId;
+    while (trimSize > 0) {
+        bool ok = iCache_->GetLast(&inodeId, &inodeWrapper, IsNotDirtyInode);
```

Comment:
Why not batch get inode?

Reply:
Still need to traverse inodes one by one, there is no difference in performance